test/pylib/rest_client.py

```diff
@@ -327,12 +327,15 @@ async def keyspace_flush(self, node_ip: str, keyspace: str, table: Optional[str]
             url += f"?cf={table}"
         await self.client.post(url, host=node_ip)
 
-    async def keyspace_compaction(self, node_ip: str, keyspace: str, table: Optional[str] = None) -> None:
+    async def keyspace_compaction(self, node_ip: str, keyspace: str, table: Optional[str] = None, aggressive_purge_tombstones: bool = False) -> None:
         """Compact the specified or all tables in the keyspace"""
-        url = f"/storage_service/keyspace_compaction/{keyspace}"
+        # actual fix in separate PR https://github.com/scylladb/scylladb/pull/20243
```

Comment:
I have fixed this in a separate PR - https://github.com/scylladb/scylladb/pull/20243 and this is here temporarily. I'll update this commit to remove the fix here as soon as https://github.com/scylladb/scylladb/pull/20243 gets merged.

Reply:
@lkshminarayanan that PR doesn't seem justified on its own. Does it actually fux anything, or us it just setting up the stage for this patch?

If it's the latter case, it should probably be included in this series.

Reply:
@bhalevy I have updated that PR #20243 now with more details. Let me know if that is sufficient.

Reply:
got it now.  It wasn't clear without the extra details (to the untrained eye)

Reply:
I have updated the patch as #20243 is now merged.